docs/guides/configuration/index.md

```diff
@@ -45,9 +45,9 @@ You can customize the following:
 - Completion (auto-completion, AI copilot, etc.)
 - Display (theme, font size, output placement, etc.)
 - Autosave 
-- Package management 
+- [Package management](https://docs.marimo.io/guides/editor_features/package_management.html#package-management) 
 - Server settings
-- VIM keybindings
+- [VIM keybindings](https://docs.marimo.io/guides/editor_features/overview.html#vim-keybindings)
```

Comment:
i think you can hyperlink by file url, then we may get errors if the file changes

Reply:
not sure if anchor's work though, could try:
`/guides/editor_features/overview#vim-keybindings`